hazelcast-helper/src/main/java/org/rutebanken/hazelcasthelper/service/HazelCastService.java

```diff
@@ -153,10 +148,30 @@ private HazelcastInstance runHazelcast(final List<String> nodes, String groupNam
         addMgmtIfConfigured(cfg);
 
         cfg.setNetworkConfig(netCfg);
+
+        getAdditionalMapConfigurations().forEach(cfg::addMapConfig);
+
         return Hazelcast.newHazelcastInstance(cfg);
     }
 
     /**
+     * Override this method if you want to provide additional configuration to the map config with name "default"
+     *
+     * @param defaultMapConfig The map config with name "default"
+     */
+    public void updateDefaultMapConfig(MapConfig defaultMapConfig) {}
+
+
+    /**
+     * If you want to provide additional map configurations to hazelcast.
+     *
+     * @return a list with desired map configurations
+     */
+    public List<MapConfig> getAdditionalMapConfigurations() {
```

Comment:
Jeg har forsøkt å gjøre dette klarere ved å lenke til hazelcast sin dokumentasjon for programatisk konfigurasjon i javadoc. Det er nå dessuten et kodeeksempel for å demonstrere hvordan det gjøres. Dersom man leser hazelcast-dokumentasjonen og ser på kodeeksempelet bør man få et inntrykk av hvordan det gjøres. Det er viktig at vi kan konfigurere maps og annen funksjonalitet i Hazelcast. Å konfigurere maps er rett rundt hjørnet i Tiamat, så jeg synes det virker unødvendig å ta denne metoden bort for å legge den til igjen. Jeg tror du mener Config#addMapConfig? Denne metoden hører til Hazelcast sitt Config-objekt, og vi bør bør ikke duplisere Hazelcast sin dokumentasjon. Derfor er både metoden og online dokumentasjon referert til i javadoc.